app/services/fixed_charge_events/create_service.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module FixedChargeEvents
+  class CreateService < BaseService
+    Result = BaseResult[:fixed_charge_event]
+
+    def initialize(organization:, subscription:, fixed_charge:, units: 0.0, timestamp: Time.current)
```

Comment:
can we only send a fixed_charge and subscription, and get organization from the subscription? (to have to pass less parameters, also we can get just organization_id form subscription to not have to load the whole organization)